src/core/core.scale.js

```diff
@@ -28,11 +28,13 @@ defaults._set('scale', {
 
 	// scale label
 	scaleLabel: {
+		// display property
+		display: false,
+
 		// actual label
 		labelString: '',
 
-		// display property
-		display: false,
+		lineHeight: 1.5
```

Comment:
@etimberg It was `1.5` previously but we could change that default for `1.2` to be more consistent?